meta.go

```diff
@@ -0,0 +1,14 @@
+package flanneltpr
+
+// Metadata to create operatorkit TPR.
+
+const (
+	// Name represents the name of the third party resource within Kubernetes.
+	Name = "flannel.network.giantswarm.io"
```

Comment:
@xh3b4sd sorry I didn't catch that earlier. Should be `flannel-network.giantswarm.io`.